src/extension/vectortiles/src/test/java/org/geotools/renderer/lite/VectorMapRenderUtilsTest.java

```diff
@@ -0,0 +1,65 @@
+package org.geotools.renderer.lite;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.awt.Rectangle;
+import java.io.IOException;
+import java.util.Arrays;
+import org.geotools.feature.FeatureCollection;
+import org.geotools.map.Layer;
+import org.geotools.styling.FeatureTypeStyle;
+import org.geotools.styling.Rule;
+import org.geotools.styling.Style;
+import org.junit.Test;
+import org.opengis.feature.type.FeatureType;
+import org.opengis.filter.expression.Expression;
+
+public class VectorMapRenderUtilsTest {
```

Comment:
In addition to this unit test, and integration test using an actual style with a transfrom would be nice, although is not strictly necessary.